### DISCUSSION TITLE
Bountysource is defunc

### DISCUSSION THREAD

**Discussion by splatch** [community/community#81460] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Please consider holding the support for bountysource in github as it is defunct since a while. The issues are unanswered and bounties seem to not get paid for developers. The app.bountysource.com doesn't work since a while.

**Comment by ashkan0201** [comment]
Response #2

**Reply by ashkan020111** [reply]
yes
